### ISSUE TITLE
[Bug] test case segwalrep/dtm_recovery_on_standby hang

### ISSUE THREAD

**Issue by smartyhero** [cloudberrydb/cloudberrydb#250]
### Cloudberry Database version

 PostgreSQL 14.4 (Cloudberry Database 1.0.0+5d6f0e9a24 build commit:5d6f0e9a247ffe695377867ba351f1f4db060210) on x86_64-pc-linux-gnu, compiled by gcc (GCC) 7.3.0, 64-bit compiled on Oct 18 2023 14:11:14 (with assert checking)

### What happened

run test case on kylin os use x86_64 cpu 
![image](https://github.com/cloudberrydb/cloudberrydb/assets/76774978/80945c42-40a4-4818-92a0-8238e3666378)

`Killed by signal 15` because the CI/CD pipeline timeoutd

### How to reproduce

rerun icw test case on kylin10 os  use x86_64 cpu

### Operating System

kylin10 x86_64 

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://cloudberrydb.org/community/coc).

**Comment by my-ship-it** [comment]
Does it happen frequently?

**Comment by smartyhero** [comment]
Yes, they must appear, and they all hang in the dtm_recovery_on_standby case  @my-ship-it

**Comment by congxuebin** [comment]
Just fyi, I was told this issue is blocking subsequent ICW test. Thus I would suggest priority of this issue could be raised.

**Comment by yjhjstz** [comment]
can you offer me the ci pipeline ?
@smartyhero
